### PR TITLE
Swap out fancy quotes for keyboard accessible quotes

### PR DESCRIPTION
**Pull request by jasonm23**
Text cleaning of special quotes.

### CODE REVIEW

File: speed-type.el [hunk 1]
```diff
@@ -398,6 +398,20 @@ are color coded and stats are gathered about the typing performance."
                             ""
                             str))
 
+(defvar speed-type--special-chars '(("“" . "\"") ("”" . "\"") ("‘" . "'") ("’" . "'"))
```
Comment (dakra) [inline speed-type.el:401]: I would make that a defcustom and maybe call it `speed-type-replace-string` or something like that.
I could imagine some people want to replace different things.
E.g. I personally would replace German Umlauts e.g. ü -> ue

Comment (jasonm23) [reply speed-type.el:401]: Sounds like a plan.  I'll push the edit shortly.

**Comment by jasonm23** [comment]
oop I didn't rename it.  one moment...

**Comment by jasonm23** [comment]
@dakra - have a look now and let me know if there's anything else to do.

**Comment by dakra** [comment]
Thanks. 

(I merged it in master but since you created this PR from your forks master branch I wasn't able to force-push to it and github doesn't recognize it as merged.)

**Pull request closed without merging**